GDWeave.Sample/GDWeave.Sample.csproj

```diff
@@ -3,6 +3,8 @@
         <TargetFramework>net8.0</TargetFramework>
         <ImplicitUsings>enable</ImplicitUsings>
         <Nullable>enable</Nullable>
+        <GDWeavePath></GDWeavePath> 
```

Comment:
This isn't supposed to be specified in the .csproj - users can set an environment variable or .user file to set this

Reply:
why, it works if you put it there
dont need to edit environment variables

Reply:
also that's the point of a PropertyGroup, to specify paths

Reply:
> why, it works if you put it there

It works, but if two developers have different paths for their game install (they probably will), they won't be able to set both paths at once in the project file (also with that just being a dumb idea).

> dont need to edit environment variables

This can also be done in Rider with `File | Settings | Build, Execution, Deployment | Toolset and Build` or with a .user file in Visual Studio.

> also that's the point of a PropertyGroup, to specify paths

This is user-specific configuration which should not be included in source control. MSBuild properties are also not used for only paths.